[ZT] Gateway policies content strategy and maintenance (#4206)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/filtering/http-policies/_index.md

| Selector | Operator | Value | Action |
| - | - | - | - |
| Content Categories | in | `Education` | Allow |

### Block

The Block action blocks outbound traffic from reaching destinations you specify within the [Selectors](#selectors) and [Value](#value) fields. For example, the following configuration blocks users from being able to upload any file type to Google Drive:

| Selector | Operator | Value | Action |
| - | - | - | - |
| Application | in | `Google Drive` | Block |
| Upload Mime Type | matches regex | `.*` |  |

### Isolate

For more information on this action, refer to the documentation on [Browser Isolation policies](/cloudflare-one/policies/browser-isolation/).

### Do Not Isolate

For more information on this action, refer to the documentation on [Browser Isolation policies](/cloudflare-one/policies/browser-isolation/).

### Do Not Inspect

{{<Aside type="Warning" header="Warning">}}

When a *Do Not Inspect* rule is created for a given hostname, application, or app type, no traffic will be inspected.

{{</Aside>}}

> *Do Not Inspect* lets administrators bypass certain elements from inspection. Administrators who wish to bypass a site must match against the host in order to prevent HTTP inspection from occurring on both encrypted and plaintext traffic.

Author: Good point. Since we assume that administrators are the ones reading the docs, then `you` would be what our style guide recommends.

> *Do Not Inspect* lets administrators bypass certain elements from inspection. Administrators who wish to bypass a site must match against the host in order to prevent HTTP inspection from occurring on both encrypted and plaintext traffic.

Author: https://wiki.cfops.it/display/Prod/People%2C+First+Person%2C+Second+Person

> *Do Not Inspect* lets administrators bypass certain elements from inspection. Administrators who wish to bypass a site must match against the host in order to prevent HTTP inspection from occurring on both encrypted and plaintext traffic.

Author: ```suggestion
*Do Not Inspect* lets you bypass certain elements from inspection. If you wish to bypass a site, your policy must match against the host in order to prevent HTTP inspection from occurring on both encrypted and plaintext traffic.
```
